notebooker/web/static/notebooker/index.js

```diff
@@ -1,132 +1,133 @@
-
-function getCurrentFolder(){
-    if (window.location.pathname.startsWith("/folder/")){
-        currentFolder = window.location.pathname.substring("/folder/".length) + "/"
-    } else {
-        currentFolder = ""
+const getCurrentFolder = () => {
+    let currentFolder = "";
+    if (window.location.pathname.startsWith("/folder/")) {
+        currentFolder = window.location.pathname.substring("/folder/".length) + "/";
     }
-    return currentFolder
-}
+    return decodeURI(currentFolder);
```

Comment:
This was the main culprit - the current folder had `%20` etc. and so the results were incorrectly being filtered out.